tensorflow_lite_support/python/task/processor/proto/embedding_pb2.py

```diff
@@ -13,8 +13,144 @@
 # limitations under the License.
 """Embedding result protobuf."""
 
+import dataclasses
+from typing import Any, List
+
+from tensorflow.tools.docs import doc_controls
 from tensorflow_lite_support.cc.task.processor.proto import embedding_pb2
 
-FeatureVector = embedding_pb2.FeatureVector
-Embedding = embedding_pb2.Embedding
-EmbeddingResult = embedding_pb2.EmbeddingResult
+_FeatureVector = embedding_pb2.FeatureVector
+_Embedding = embedding_pb2.Embedding
+_EmbeddingResult = embedding_pb2.EmbeddingResult
+
+
+@dataclasses.dataclass
+class FeatureVector:
+  """A dense feature vector. Only one of the two fields is ever present.
+  Feature vectors are assumed to be one-dimensional and L2-normalized.
+
+  Attributes:
+    value_float: Raw output of the embedding layer. Only provided if `quantize`
+      is set to False in the `EmbeddingOptions`, which is the case by default.
+    value_string: Scalar-quantized embedding. Only provided if `quantize` is
+      set to true in `EmbeddingOptions`.
+  """
+
+  value_float: List[float] = None
```

Comment:
Due to more flexibility in Python, I think we can use a single attribute `value: np.ndarray` here. The datatype of elements in the array can be either np.single (float) or np.byte. You can look at the data type to convert it to protobuf's value_float and value_string accordingly.